p/scripts/user.query.js

```diff
@@ -11,27 +11,63 @@ function init_draggable() {
 		return;
 	}
 	
-	let source;
+	var source;
```

Comment:
Any reason for `var` ? (just being curious)

Reply:
P.S. This changes the scope to global

Reply:
Looks accidental to me.